### ISSUE TITLE
Feed-parser response does not expose Date header for CORS responses

### ISSUE THREAD

**Issue by alex-deaconu** [Rise-Vision/rise-data-rss#17]
This is causing RSS component caching to not work correctly.

To confirm, Cache depends on `date` header comparison to check for `refresh`/`expiry`. If the header is not exposed it cannot perform that comparison, and in this manner Cached responses are not used for `refresh` (the component sets `expiry=-1` which skips that comparison).

As per spec, CORS responses only "safelist" the following headers as part of the response:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers

Basically, if we want the response to contain the Date header, we need to have the following header definition:
`access-control-expose-headers: date`
as part of the Feed Parser response.

This header definition is already available from `proxy.risevision.com`.

Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu this has been handled as part of this PR: https://github.com/Rise-Vision/feed-parser/pull/25

Please re open the issue if needed!

@stulees FYI
